comp/dogstatsd/packets/telemetry.go

```diff
@@ -52,3 +58,17 @@ func InitTelemetry(buckets []float64) {
 		"Time in nanoseconds to push a packets from a listeners to dogstatsd pipeline",
 		buckets)
 }
+
+// TelemetryTrackPackets tracks the number of packets in the channel and the number of bytes
+func TelemetryTrackPackets(packets Packets, listenerID string) {
+	tlmChannelSizePackets.Add(float64(len(packets)), listenerID)
+	tlmChannelSizePacketsBytes.Add(float64(packets.SizeInBytes()+packets.DataSizeInBytes()), listenerID)
+}
+
+// TelemetryUntrackPackets untracks the number of packets in the channel and the number of bytes
+func TelemetryUntrackPackets(packets Packets) {
+	for _, packet := range packets {
```

Comment:
Would it make sense for this function to use Packets#SizeInBytes, like the `TelemetryTrackPackets`?

Reply:
This could not work because packets are not guaranteed to belong to the same listenerId.

Reply:
Why not? Isn't it exactly the same `packets` object we handle in `TelemetryTrackPackets`, just on the other side of the channel?

Reply:
`Packets` is just a slice of `Packet`
- when we call `TelemetryTrackPackets(packets, listenerId)` the caller tells us that it's known that all sub-packet have the same value for `listenerId`
- when we call `TelemetryUnTrackPackets(packets)` something might have merged `Packets` together and we don't know about it